src/Ocelot/Middleware/OcelotMiddlewareExtensions.cs

```diff
@@ -53,6 +53,9 @@ public static class OcelotMiddlewareExtensions
         {
             await CreateAdministrationArea(builder);
 
+            // Initialises downstream request
+            builder.UseDownstreamRequestInitialiser();
+
             // This is registered to catch any global exceptions that are not handled
             builder.UseExceptionHandlerMiddleware();
```

Comment:
Probably missing something but shouldn't this be the first piece of middleware, or are we assuming the UseDownstreamRequestInitialiser cannot throw an exception and if it does it will be caught!